webpack/assets/javascripts/react_app/redux/API/__tests__/APISelectors.test.js

```diff
@@ -31,6 +32,13 @@ const failureState = {
   },
 };
 
+const withTimestamp = {
+  API: {
+    [key]: {
+      payload: { timestamp: 10000 }
```

Comment:
Timestamp 10000? :thinking: shouldn't that be a date instead?
I'd rather have some real value here if possible...